src/vs/workbench/contrib/chat/browser/chatDragAndDrop.ts

```diff
@@ -489,16 +491,18 @@ function symbolId(resource: URI, range?: IRange): string {
 	return resource.fsPath + rangePart;
 }
 
-// function extractImageAttributes(html: string): { src: string; alt?: string } {
-// 	const imgTagRegex = /<img[^>]+src=["']([^"']+)["'][^>]*>/;
-// 	const altRegex = /alt=["']([^"']+)["']/;
+async function extractImageAttributes(e: DragEvent): Promise<string | Uint8Array> {
+	const files = e.dataTransfer?.files;
+	if (files && files.length > 0) {
```

Comment:
Does this need to check the mime type of the `file` to find the first `image/*` file?